src/java/api.js

```diff
@@ -21,4 +23,9 @@ module.exports = class JavaRealmAPI extends RealmAPI {
   async changeRealmState (realmId, state) {
     return await this.rest.put(`/worlds/${realmId}/${state}`)
   }
+
+  async getRealmWorldDownload (realmId, slotId) {
```

Comment:
I'm confused as to why `Download` can't be inlined into `Backup`? Is this getRealmWorldDownload() not using backups and calling a seperate API?

Reply:
Yeah that's correct, on Java you cannot download individual backups only the world at the time the request is called.